sdk/tests/wallet/output_preparation.rs

```diff
@@ -608,12 +611,13 @@ async fn prepare_output_remainder_dust() -> Result<()> {
     request_funds(&wallet_0).await?;
     request_funds(&wallet_1).await?;
 
-    let rent_structure = wallet_0.client().get_rent_structure().await?;
-    let token_supply = wallet_0.client().get_token_supply().await?;
+    let storage_score_params = wallet_0.client().get_storage_score_parameters().await?;
 
     let balance = wallet_0.sync(None).await?;
-    let minimum_required_storage_deposit =
-        MinimumStorageDepositBasicOutput::new(rent_structure, token_supply).finish()?;
+    let minimum_required_storage_deposit = BasicOutputBuilder::new_with_amount(0)
+        .add_unlock_condition(AddressUnlockCondition::new(wallet_1.address().await))
+        .finish()?
+        .minimum_amount(storage_score_params);
```

Comment:
There is a `minimum_storage_deposit` in the output mod.rs, I'm wondering if we should move it as a static method to BasicOutput or something. This just seems like a pattern we may be doing quite a lot